### DISCUSSION TITLE
How to trigger an action when a form tab is clicked?

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#5576] [Help]
Hello,

Looking for help on how to trigger an action when a specific form tab is clicked by a user

e.g. in below form, there is a tab for Customers, Orders, and Notes
https://serenity.is/demo/Northwind/Customer#edit/1

I would like to set up the code so that when a user clicks the Orders tab, it triggers an automatic Refresh of the the embedded Orders grid.

**Comment by muhammedyaman** [comment]
In the Northwind.CustomerDialog.Template.html file
replace this:
`<li><a href="#~_TabOrders"><span>{{text:"Db.Northwind.Order.EntityPlural"}}</span></a></li>`
with this: 
`<li><a href="#~_TabOrders" onclick="javascript: GridRefresh()"><span>{{text:"Db.Northwind.Order.EntityPlural"}}</span></a></li>`

the rest is javascript.
